### ISSUE TITLE
KCL sandbox submission review

### ISSUE THREAD

**Issue by joshgav** [cncf/tag-app-delivery#430]
[KCL](https://kcl-lang.io/) has been [proposed to CNCF sandbox](https://github.com/cncf/sandbox/issues/48). This issue is to track discussions and reviews of KCL to help with deciding whether to accept it in CNCF sandbox.

We've asked @Peefy and team to join an upcoming TAG meeting to discuss the following about the project:

- the values KCL proposes to end users
- KCL's high-level technical architecture
- the project's near-term roadmap
- state of the project's community and governance
- comparison with existing projects (e.g. helm, kustomize/kpt, ytt)

**Comment by joshgav** [comment]
Thank you @Peefy for presenting at the last TAG meeting.

- Notes are here: <https://docs.google.com/document/d/1OykvqvhSG4AxEdmDMXilrupsX2n1qCSJUWwTc3I7AOs/edit#heading=h.hvn2d1nejnj5>
- Recording is here: <https://youtu.be/nZcMMluQF9g?t=956>

In this initial review we learned that KCL provides structured configuration templating and rendering, complementing the likes of CNCF projects like Helm, kpt, ytt (Carvel) and even Crossplane XRM. We also learned of some significant companies that are using KCL in China. From these perspectives it's a good fit for CNCF Sandbox.
